packages/@okta/i18n/src/properties/login.properties

```diff
@@ -983,3 +983,6 @@ oie.browser.error.NotAllowedError = The operation either timed out or was not al
 # soft_token API errors
 authfactor.challenge.soft_token.invalid_passcode = Your code doesn't match our records. Please try again.
 authfactor.challenge.soft_token.used_passcode = Each code can only be used once. Please wait for a new code and try again.
+
+# rate limit
+tooManyRequests = Too many requests
```

Comment:
Can we add an `oie/idx` prefix to this key ?. It is a format that we follow for all oie keys. We are also looking to add some automation for customization where we need to know if a key is an oie key or not.

Reply:
I double checked the error and look like `tooManyRequests` in the currently format is mainly used for OIE. 

I have added `oie` prefix to this key. The V1 error look more like error in `error-429-api-limit-exceeded.json`.